torch/_inductor/kernel/flex/flex_flash_attention.py

```diff
@@ -171,20 +171,34 @@ def _use_flex_flash_attention(
     mask_graph: Subgraph,
     kernel_options: dict[str, Any],
     num_score_mod_placeholders: int,
+    force_impl: Literal["DEFAULT", "TRITON", "FLASH", "DECODE"],
```

Comment:
Slght nit: literal is now defined in two different files. May want ot consolide somewhere as an internal TypeAlias